app/views/pages/downloads.ctp

```diff
@@ -85,6 +85,7 @@ $this->set('title_for_layout', $pages->formatTitle(__('Download sentences', true
         $field_struct_str = __('Fields and structure', true);
         $file_desc_str = __('File description', true);
         $sent_id_str = __('Sentence id', true);
+        $mark_id_str = __('Sentence id', true);
```

Comment:
The name of the variable is not appropriate because it’s not an id. Use e.g. `$mark_str` instead.
The string is incorrect, it should be something like 'Mark'.